index.html

```diff
@@ -8,14 +8,30 @@
 
   <div>	
     <div >
-      <h1>The OBO Foundry</h1>
+      <h1>The Open Biological and Biomedical Ontology (OBO) Foundry<h1>
+        <h3>Community development of interoperable ontologies for the biological sciences</h3>
```

Comment:
Has this subheading been approved by someone other than us?

Reply:
@nlharris

Reply:
The proposed new home page content was discussed during the last OBO Operations meeting and there were some suggestions on it (which I responded to) but no one specifically commented on that text (which I believe was @mellybelly's invention). The general consensus was that the overall revisions looked good, and that I should go ahead and make a PR and we could continue to refine the wording. I welcome feedback!